controllers/resources/bentorequest_controller.go

```diff
@@ -1756,32 +1767,54 @@ if [ -f "{{.ModelDirPath}}/.exists" ]; then
 	exit 0
 fi
 
+if [ -n "{{.HfToken}}" ]; then
+	echo "HF_TOKEN is set, using HF_TOKEN to download model..."
+	export HF_TOKEN="{{.HfToken}}"
```

Comment:
‌‌‌‌‌Placing the token in plain text within the script content is also unsafe.

Reply:
@yetone What's the best way of transmitting the token and set the env var? Do we directly load the token from the source file here?

Reply:
@rickzx 

1. Store tokens as key-value pairs in the secret
2. Reference this secret using env from

https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/#configure-all-key-value-pairs-in-a-secret-as-container-environment-variables

Reply:
Thanks! fixed